cloudant-client/src/main/java/com/cloudant/client/api/Changes.java

```diff
@@ -73,8 +73,8 @@
  *
  * @author Ganesh K Choudhary
  * @see ChangesResult
- * @see <a target="_blank" href="https://docs.cloudant.com/database.html#get-changes">Databases -
- * get changes</a>
+ * @see <a href="https://console.bluemix.net/docs/services/Cloudant/database.html#get-changes"

```

Comment:
This is missing an `/api/` path element.

Reply:
Fixed in 7a6c69c.